.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 docs
 .tmp
+app/lib
```

Comment:
@gdomorski These errors are here because these files need to have an extra line at the bottom of the file. You can set this up in sublime to add this extra line automatically in the sublime settings.